samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    calgary_no_southland_solar_layer_sdf = calgary_no_southland_solar_layer.query().sdf
</pre><p>better to split the cell before this line, so the map preview will show.</p><p>Can you take a screenshot of the map and add it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>